language-extensions/java/build/linux/build-java-extension.sh

```diff
@@ -66,9 +66,9 @@ function build {
 	# This will create the java extension package with unsigned binaries, this is used for local development and non-release builds. release
 	# builds will call create-java-extension-zip.sh after the binaries have been signed and this will be included in the zip
 	#
-	zip ${TARGET}/java-lang-extension libJavaExtension.so.1.0
+	tar -czvf ${TARGET}/java-lang-extension.tar.gz libJavaExtension.so.1.0
```

Comment:
if we do this, we need also update build pipeline in the internal repo.

Reply:
so, I would suggest, using the git submodule from this private branch, in the internal repo, we generate the nuget and ensure everything is as expected.

Reply:
reverted these changes so pipelines will work as-is.